automation/src/test/resources/startTunnel.sh

```diff
@@ -7,8 +7,14 @@
 
 # Outputs: a file with the name tunnel_$PORT.pid with the system pid of the tunnel process
 
+echo "starting tunnel bastion ${BASTION_NAME} with resource id ${RESOURCE_ID} in resource group ${RESOURCE_GROUP} on port ${PORT}"
+
+nohup az network bastion tunnel --name "$BASTION_NAME" --resource-group "$RESOURCE_GROUP" --target-resource-id "${RESOURCE_ID}" --resource-port 22 --port $PORT > /tmp/nohup.out &
+
+echo "displaying nohup output"
+cat /tmp/nohup.out
+echo "done displaying nohup output"
 
-nohup az network bastion tunnel --name "$BASTION_NAME" --resource-group "$RESOURCE_GROUP" --target-resource-id "${RESOURCE_ID}" --resource-port 22 --port $PORT > /dev/null 2>&1 &
```

Comment:
before, 
- this reads 'send the stdout of this command into /dev/null' (the ` > /dev/null`), 
- then `2>&1` means 'also send stderr where you sent stdout'.
- the final `&` runs it as a background process (i.e. allowing the script to continue execution)